src/injected/utilityScript.ts

```diff
@@ -35,6 +35,10 @@ export default class UtilityScript {
     return serializeAsCallArgument(value, (value: any) => ({ fallThrough: value }));
   }
 
+  previewNode(node: Node): string {
```

Comment:
The split between the injectedScript and utilityScript was that the latter could not know about the DOM. Otherwise, it is the same thing...

Reply:
Done.